cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -878,6 +878,10 @@ class HydratorPlusPlusTopPanelCtrl {
   }
 
   _checkAndShowConfirmationModalOnActionPlugin(proceedCb) {
+    if (!this.HydratorPlusPlusConfigStore.validateState(true)) {
```

Comment:
I don't remember, but does the validateState will actually trigger an error banner? If it doesn't, we might need to do that, otherwise the experience will be the user clicking the button but nothing happens

Reply:
Yes, with `true` it does.
![image](https://user-images.githubusercontent.com/2728821/102561384-6a1e4d00-4089-11eb-966b-fbeb29172e91.png)